js/mandala.js

```diff
@@ -0,0 +1,411 @@
+        return response;
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        let requestSide = undefined;
+        let requestType = undefined;
+        if (side in this.sides) {
+            requestSide = this.sides[side];
+        } else {
+            throw new NotSupported ('Side ' + side + ' not supported');
+        }
+        if (type in this.types) {
+            requestType = this.types[type];
+        } else {
+            throw new NotSupported ('Type ' + type + ' not supported.');
+        }
+        const method = 'signedPostOrders';
+        await this.loadMarkets ();
+        const request = {
+            'symbol': this.markets[symbol]['id'],
+            'side': requestSide,
+            'type': requestType,
+            'quantity': amount,
+        };
+        if (price !== undefined) {
+            request['price'] = price;
+        }
+        const response = await this[method] (this.extend (request, params));
+        return response;
+    }
+
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        const method = 'signedGetOrdersDetail';
+        const request = {
+            'orderId': id,
+        };
+        const response = await this[method] (this.extend (request, params));
+        return response;
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        const method = 'signedPostOrdersCancel';
+        const request = {
+            'orderId': id,
+        };
+        const response = await this[method] (this.extend (request, params));
+        return response;
+    }
+
+    async fetchBalance (params = {}) {
+        const method = 'signedGetAccountSpot';
+        const response = await this[method] (params);
+        return response;
+    }
+
+    async fetchTrades (symbol, since = undefined, limit = undefined, params = {}) {
+        let method = 'binanceGetV3Trades';
+        await this.loadMarkets ();
+        let marketSymbol = this.markets[symbol]['id'];
+        if (marketSymbol.indexOf ('MDX') >= 0) {
+            method = 'openGetMarketTrades';
+        } else {
+            marketSymbol = marketSymbol.replace ('_', '');
+        }
+        const request = {
+            'symbol': marketSymbol,
+        };
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        let response = await this[method] (this.extend (request, params));
+        if (marketSymbol.indexOf ('MDX') >= 0) {
+            response = this.safeValue (response.data, 'list');
+        }
+        const result = [];
+        for (let i = 0; i < response.length; i++) {
+            result.push (this.convertTrade (symbol, response[i]));
+        }
+        return result;
+    }
+
+    async fetchMyTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired ('fetchMyTrades requires a symbol argument');
```

Comment:
Can you as well add examples of symbols required, it makes for a clearer debugging.

Reply:
@mrbaguvix usually this is about all unified symbols, as documented in the Manual: https://github.com/ccxt/ccxt/wiki/Manual#symbols-and-market-ids